src/renderer/components/+workloads-deployments/deployment-menu.tsx

```diff
@@ -0,0 +1,47 @@
+/**
```

Comment:
Note: This component was extracted to separate file to avoid circular dependency